File: 21560 - Sai Suraj/week 6/215_Kth Largest Element in an Array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        minh = []
        for i in nums:
            heappush(minh,i)
            if len(minh)>k:
                heappop(minh)
        return minh[0]

File: 21558 - Nithin/week 6/215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int> pq;
        int n = nums.size();
        if(n == 1)
            return nums[0];
        
        for(int i = 0;i < n;i++)
            pq.push(nums[i]);
        int c = 0;
        while(c < k-1){
            pq.pop();
            ++c;
            }
        return pq.top();
    }
};